{{cookiecutter.pipeline_slug}}/nextflow.config

```diff
@@ -12,6 +12,7 @@
 params {
   // Pipeline version
   version = '{{ cookiecutter.version }}'
+  container = 'nfcore/"{{ cookiecutter.pipeline_name }}"' // Container slug. Stable releases should specify release tag!
```

Comment:
We have a specific cookiecutter template for this:
```nextflow
container = '{{ cookiecutter.dockerhub_slug }}'
```

See [`cookiecutter.json`](https://github.com/nf-core/cookiecutter/blob/master/cookiecutter.json) for the list of variables.